lib/msf/core/post/windows/shadowcopy.rb

```diff
@@ -182,6 +182,25 @@ def start_vss
         return false
       end
     end
+    unless start_swprv
+      return false
+    end
+    return true
+  end
+
+  def start_swprv
+    vss_state = wmic_query('Service where(name="swprv") get state')
```

Comment:
Ah yes sorry it is a post method :). But `wmic_query` is also writing to clipboard or writing to file, with equivalent reads to both. If you check the source it is doing about 4-5 calls under the hood.